### ISSUE TITLE
link prefetch appears to show-up as double image download on "page weight"

### ISSUE THREAD

**Issue by a701440** [YellowLabTools/YellowLabTools#227]
We have link rel="prefetch" href="..." in the head for a large image that is also used lower on the page. On the page weight report it appears that the image is counted two times and lowers the score.

**Comment by gmetais** [comment]
Hi,

I'm a bit surprised. Could you please check if the request's caching headers are correct? No "no-cache" and a "max-age" > 0 ?

And share the page url if it's not private!

**Comment by a701440** [comment]
https://www.kayak.com/hotels 
with cookie features=horizonleancomp

**Comment by gmetais** [comment]
Thanks for the url.
It looks like PhantomJS records two different requests and this is a bug. I'll have a look into this when I have a bit more time!

**Comment by gmetais** [comment]
Hi @a701440,

It looks like your `prefetch` link got recently replaced by `preload`. Preload is more relevant in your use case, but it's not supporter by the version of PhantomJS (v2.1) in use in YLT.

I keep this bug about `prefetch` open and I'll try to reproduce it on test webpage.

**Comment by gmetais** [comment]
Tested (finally) on a very simple page:

```html
<html>
<head>
    <title>Page with a prefetched image</title>
    <link rel="prefetch" href="png-image.png">
</head>
<body>
    <img src="png-image.png">
</body>
</html>
```

Phantomas detects two different images. Let's wait for the next version that should run on Chrome's headless mode.

**Comment by gmetais** [comment]
Fixed by #327.
